### PR TITLE
[patch] update version to v2

### PR DESCRIPTION
**Pull request by kyfujisa**
* [x] Version of the authorization-proxy in go.mod(and code)
  *  same #53

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/yahoojapan/authorization-proxy
+module github.com/yahoojapan/authorization-proxy/2
```
Comment (WindzCUHK) [inline go.mod:1]:
```suggestion
module github.com/yahoojapan/authorization-proxy/v2
```
should be `v2`, just `2` does not work.
The `v` prefix is required for golang to treat it as version.
https://research.swtch.com/vgo-import

Comment (WindzCUHK) [reply go.mod:1]: flow:
1. merge this PR
1. `2.0.1` released by pipeline
1. tag `v2.0.1` and release manually

Comment (kyfujisa) [reply go.mod]: OK, I  fixed it according to this.
9aee247

**Comment by WindzCUHK** [comment]
LGTM

**Pull request merged**